settings.gradle.kts

```diff
@@ -25,9 +25,9 @@
  */
 pluginManagement {
     repositories {
-        maven("https://dl.bintray.com/kotlin/kotlin-eap")
+        mavenLocal()
```

Comment:
I guess maven local should not be here. It's usually a bad idea to rely on local repo.